internals/reminders.py

```diff
@@ -148,10 +188,13 @@ class FeatureAccuracyHandler(AbstractReminderHandler):
   """Periodically remind owners to verify the accuracy of their entries."""
 
   ACCURACY_GRACE_PERIOD = timedelta(weeks=4)
+  ALLOW_ESCALATION = True 
+  # Notifications after the 2nd with no acknowledgement will be escalated.
+  ESCALATION_CHECK = lambda self, f: f.outstanding_notifications >= 2
```

Comment:
Why not define a method in the base class that always returns False and then override that method here?  Then you would not need ALLOW_ESCALATION.

Reply:
My thought process was that if we might have different implementations of the method that take different arguments, but even then, my implementation is not capable of passing in different arguments here, so your suggestion would be better overall! I've made the change. 🙂